repo/packages/B/beta-mesosphere-jupyter-service/0/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "name": "beta-mesosphere-jupyter-service",
+  "version": "1.3.0-0.35.4-beta",
+  "scm": "https://github.com/dcos-labs/dcos-jupyterlab-service",
+  "maintainer": "support+mesosphere-jupyter-service@mesosphere.com",
+  "website": "https://docs.mesosphere.com/services/beta-jupyter",
```

Comment:
services page shouldn't already be live. Also, the services page seems inconsistent with this release.

Reply:
True, this will eventually be updated to reflect all the changes in this package. We need to get this out there soon since this newer package has a *lot* of improvements that users have been waiting on.